test/pom.xml

```
@@ -31,6 +35,77 @@
 			<groupId>org.projectlombok</groupId>
 			<artifactId>lombok</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc${oracle.driver.jdkversion}</artifactId>
+			<version>${oracle.driver.version}</version>
+		</dependency>
```

```suggestion
		<dependency>
			<groupId>com.oracle.database.jdbc</groupId>
			<artifactId>ojdbc${oracle.driver.jdkversion}</artifactId>
			<version>${oracle.driver.version}</version>
			<scope>runtime</scope>
		</dependency>
```
Mis ik iets of ben je hier de scope vergeten?

---

Yep Vergeten